### PR TITLE
issue-1-create-an-entry (Code Journal) - Ken Kieu

### PR DESCRIPTION
**Pull request by kenkieu**
Completed issue-1-create-an-entry.

issue-1-create-an-entry-responsive
issue-1-create-an-entry-localstorage

### CODE REVIEW

**Comment by kenkieu** [comment]
Persistence after refreshes
persistence-after-refresh

File: index.html [hunk 1]
```diff
@@ -1,25 +1,64 @@
...
+          <div class="column-half">
+            <img src="images/placeholder-image-square.jpg" alt="user-image" class="width-100">
+          </div>
+          <div class="column-half">
+            <label for="title" class="bold med-text">Title</label>
+            <input required type="text" id="title">
+            <label for="photo-url" class="bold med-text">Photo URL</label>
+            <input required type="text" id="photo-url">
+          </div>
+          <div class="row width-100">
```
Comment (codingcodymiller) [inline index.html]: You can just remove this element, and let the `column-full` child element just be in the same `row` as the above `column-half` elements.

File: index.html [hunk 2]
```diff
@@ -1,25 +1,64 @@
...
+            <div class="column-full">
+              <label for="notes" class="bold med-text">Notes</label>
+              <textarea required type="text" id="notes" cols="30" rows="10"></textarea>
+            </div>
+          </div>
+          <div class="row width-100">
```
Comment (codingcodymiller) [inline index.html]: Same thing here, these can all be in the same `row` since the `row` class is set to `flex-wrap: wrap`

File: js/main.js [hunk 3]
```diff
@@ -1,2 +1,29 @@
...
+  entry.title = $form.elements[0].value;
+  entry.photoUrl = $form.elements[1].value;
+  entry.note = $form.elements[2].value;
+  entry.entryId = data.nextEntryId;
+  data.nextEntryId++;
+  data.entries.unshift(entry);
+  $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $form.elements[0].value = '';
+  $form.elements[1].value = '';
+  $form.elements[2].value = '';
```
Comment (codingcodymiller) [inline js/main.js]: You should just use the `reset` method

https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset

File: js/main.js [hunk 4]
```diff
@@ -1,2 +1,29 @@
...
+var $photoUrl = document.querySelector('#photo-url');
+var $img = document.querySelector('img');
+$photoUrl.addEventListener('input', handlePhoto);
+
+function handleSubmit(event) {
+  event.preventDefault();
+  var entry = {};
+  entry.title = $form.elements[0].value;
+  entry.photoUrl = $form.elements[1].value;
+  entry.note = $form.elements[2].value;
```
Comment (codingcodymiller) [inline js/main.js]: Please use the named properties rather than the numbered indexes, it is easier to read and less error-prone

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**